sdk/spring/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 4.1.0-beta.1 (Unreleased)
 
 ### Features Added
+- GA the `spring-cloud-azure-starter-keyvault-certificates`. 
 
 ### Breaking Changes
 
```

Comment:
Shall we add some description for these changes here?

Reply:
Any suggestions on which aspect should we add for?

Reply:
They have the same library name with different functions, I recommend we can add some description to tell uses the difference, it's a new starter.

Reply:
It's called azure-spring-boot-starter-keyvault-certificates in the 3.x

Reply:
But we can add the feature it supports.